contracts/etf/examples/deploy.rs

```diff
@@ -0,0 +1,46 @@
+use std::sync::Arc;
+
+use abstract_boot::{
+    boot_core::networks::{parse_network, NetworkInfo},
+    boot_core::*,
+    AppDeployer,
+};
+use semver::Version;
+
+use abstract_etf_app::{boot::ETF, ETF_ID};
+use clap::Parser;
+use tokio::runtime::Runtime;
+
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+fn deploy_etf(network: NetworkInfo) -> anyhow::Result<()> {
+    let version: Version = CONTRACT_VERSION.parse().unwrap();
+    let rt = Arc::new(Runtime::new()?);
+    let options = DaemonOptionsBuilder::default().network(network).build();
+    let (_sender, chain) = instantiate_daemon_env(&rt, options?)?;
+    let mut etf = ETF::new(ETF_ID, chain);
+
+    etf.deploy(version)?;
+    Ok(())
+}
+
+#[derive(Parser, Default, Debug)]
+#[command(author, version, about, long_about = None)]
```

Comment:
I see you adding this to every module, have you thought about how we might be able to easily include it in boot?